### DISCUSSION TITLE
unable to get the data

### DISCUSSION THREAD

**Discussion by devopsmins** [learndevopsonline/learndevopsonline#1667] [Q&A] [answered]
Good evening
In react app when I am trying to add the data it is not showing the added data or previous data.
![project](https://github.com/learndevopsonline/learndevopsonline/assets/150717333/9db1577d-0a45-459e-b3fa-f7e2a2cb2809)

**Comment by sreedharm07** [comment]
@devopsmins 
try now its working now 
i just did backend server changes, in backend.service file unnessesary space i removed there

**Reply by devopsmins** [reply]
still having the same problem
![backend](https://github.com/learndevopsonline/learndevopsonline/assets/150717333/48125318-86e2-4537-a1ff-fa929f1105ce)

**Reply by sreedharm07** [reply]
can you ping your workstation ip

**Reply by devopsmins** [reply]
34.230.60.29

**Reply by sreedharm07** [reply, marked as answer]
Its working fine just remove this space 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/633b7ec7-e2f4-49a5-bd3e-ce7b68c51ad5)![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/597ca826-3846-4c29-8cd1-ede783cf472d)

**Reply by devopsmins** [reply]
got it...Thank you so much
